src/services/replyService.js

```diff
@@ -60,26 +54,26 @@ class ReplyService {
         return this.findOrCreateReply(replyObj, commenter);
       })
       .catch(err => console.log("trouble adding replies to db", err));
-  }
+  },
 
-  findOrCreateReply(replyObj, commenter) {
+  findOrCreateReply: function(replyObj, commenter) {
     return ReplyModel
       .findOrCreate({
         where: {id: replyObj.id},
         defaults: { ...replyObj, userId: commenter.id }
       })
       .spread((replyInOurDb, created) => {
         if (replyInOurDb.children > 0 && created) {
-          return this.fetchRepliesFromSteemit({ parentReply: replyInOurDb });
+          return this.fetchRepliesFromSteemit(replyInOurDb.postId, null, null, { parentReply: replyInOurDb });
```

Comment:
For future reference, if a function receives 2 or more arguments, it's good practice to receive the params as an object and deconstruct that object in the function so we don't have to worry about order of arguments, or empty values ie: the `null` values we're passing in here. 
If you want to refactor, it's up to you. I'm ok with this for now